Makefile

```diff
@@ -19,6 +19,11 @@ NXLIBDIR    ?= $(PREFIX)/lib/nx
 X2GOLIBDIR  ?= $(PREFIX)/lib/x2go
 CONFIGURE   ?= ./configure
 
+NX_VERSION_MAJOR=$(shell cat VERSION | cut -d"." -f1)
+NX_VERSION_MINOR=$(shell cat VERSION | cut -d"." -f2)
+NX_VERSION_MICRO=$(shell cat VERSION | cut -d"." -f3)
+NX_VERSION_PATCH=$(shell cat VERSION | cut -d"." -f4)
+
```

Comment:
Maybe it's better to use a "real" shell script here which fetches the version number part and, if it is unset, defaults to "0", to always have the parts well-defined?